### PR TITLE
User profile feed filters bookmarks

### PR DESCRIPTION
**Pull request by hrishi82**
Added:
- bookmarks page
- explore page
- user profile

- Refactored CSS

### CODE REVIEW

File: src/backend/db/users.js [hunk 1]
```diff
@@ -8,15 +8,16 @@ import { formatDate } from "../utils/authUtils";
 export const users = [
   {
     _id: "user1-libkjefbvjabvkajhfbe",
-    firstName: "Adarsh",
-    lastName: "Balika",
-    username: "adarshbalika",
+    firstName: "Panda",
```
Comment (Parul1999) [inline src/backend/db/users.js:11]: Can have more meaningful names .

File: src/components/Aside/LeftAsideBar/LeftAsideBar.jsx [hunk 2]
```diff
@@ -114,37 +113,37 @@ export const LeftAsideBar = () => {
       </ul>
 
       <ul className="sidebar-list-container sidebar-list-container-bottom">
-        <li className="sidebar-li-item">
+        {/* <li className="sidebar-li-item">
```
Comment (Parul1999) [inline src/components/Aside/LeftAsideBar/LeftAsideBar.jsx:116]: Can remove the commented code

File: src/pages/authpage/signup/SignupPage.jsx [hunk 3]
```diff
@@ -1,50 +1,52 @@
...
-          }
+          dispatch(signupHandler({
+            username: signupForm.username,
+            password: signupForm.password,
+            firstName: signupForm.firstName,
+            lastName: signupForm.lastName,
+          }))
+          
         } catch (error) {
           console.error(error);
```
Comment (Parul1999) [inline src/pages/authpage/signup/SignupPage.jsx:40]: Can have the toastr error instead of console.

**Review by rahulbakshi4 — APPROVED** [review]
Changes looks good to me. 
Suggestion: add a preview

can be merged

**Pull request merged**